dm/ctl/common/config.go

```diff
@@ -63,7 +64,7 @@ func NewConfig() *Config {
 
 	fs.BoolVar(&cfg.printVersion, "V", false, "Prints version and exit.")
 	fs.StringVar(&cfg.ConfigFile, "config", "", "Path to config file.")
-	fs.StringVar(&cfg.MasterAddr, "master-addr", "", "Master API server address, this parameter is required when interacting with the dm-master")
+	fs.StringVar(&cfg.MasterAddr, "master-addr", "", "Master API server address, this parameter is required when interacting with the dm-master, you can use environment var 'DM_MASTER_ADDR' to pass value to this flag")
```

Comment:
don't change flag comment because env and command line parameter are not the same thing 🤔 you only need to change the error message